src/mlpack/core/data/save.hpp

```diff
@@ -124,10 +121,15 @@ template<typename eT>
 bool Save(const std::vector<std::string>& files,
           arma::Mat<eT>& matrix,
           ImageInfo& info,
-          const bool fatal = false,
-          const bool transpose = true);
+          const bool fatal = false);
 
-#endif // HAS_STB.
+/**
+ * Helper function to save files.  Implementation in save_image.cpp.
+ */
+bool SaveImage(const std::string& filename,
+               arma::Mat<unsigned char>& image,
+               ImageInfo& info,
+               const bool fatal);
```

Comment:
why removed default value ?

Reply:
Ahh, oversight.  This function won't be called by a user (it's used internally by `Save()`), but it doesn't hurt to add the default value back.  Done in 6f7f91606.

Reply:
My PR broke badly :), I am trying to resurrect transpose option, Since My test were passing

Reply:
@rcurtin , Does this comment still holds good ? https://github.com/mlpack/mlpack/pull/2019#discussion_r393384919

Reply:
@jeffin143 if you like, I can spend some time with #2019 tonight and propagate my changes, just let me know.

Reply:
I will let you know by night, I am currently on it :)